Source/gg2/Objects/InGameElements/Character.events/Begin Step.xml

```diff
@@ -272,7 +284,8 @@ if(place_free(x,y+1)) // no obstacle ground below us
         // Since the inital frame of gravity is half what it used to be, the strength of jumping needs to change the same amount, which I did elsewhere.
         // I used the 30fps value of "initial frame of gravity", since that represents gg2's canonical jump arc.
         // This change PROBABLY makes explosive jumping slightly more powerful, because the initial strength of that cannot be tweaked without also altering mid-air arcs.
-        applied_gravity += 0.6*global.delta_factor/2;
+        // Actually, because GG2 used to not apply on the first frame after jumping, jumping velocity is slightly larger, not smaller.
```

Comment:
Which is completely irrelevant since you changed it, so this comment would serve only to confuse. Please remove ;)